front/app/api/project_reviews/useRequestProjectReview.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+
+import fetcher from 'utils/cl-react-query/fetcher';
+
+import projectReviewKeys from './keys';
+import { ProjectReview } from './types';
+
+const requestProjectReview = async (projectId: string) =>
+  fetcher<ProjectReview>({
+    path: `/projects/${projectId}/review`,
+    action: 'post',
+    body: {},
+  });
+
+const useRequestProjectReview = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation<ProjectReview, CLErrors, string>({
+    mutationFn: requestProjectReview,
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: projectReviewKeys.items() });
```

Comment:
Same question here - could we invalidate a specific item instead? Or do we need to invalidate all items?